update the tutorial based on the perception package update (#126)
Unity-Technologies/Unity-Robotics-Hub: tutorials/pose_estimation/Documentation/4_pick_and_place.md

* **Action**: On the `Simulation Scenario` GameObject, uncheck the `Fixed Length Scenario` component to disable it, as we are no longer in the Data Collection phase. If you want to collect new data in the future, you can always check back on the `Fixed Length Scenario` and uncheck to disable the `ROSObjects`. 

> * **Action**: On the `Main Camera` GameObject, uncheck the `Perception Camera` script component. This will disable highlighted objects' edge due to their labeling (in our case only the cube). 

Reviewer: ```suggestion
* **Action**: On the `Main Camera` GameObject, uncheck the `Perception Camera` script component, since we do not need it anymore. 
```
